### ISSUE TITLE
Graphs: hop-specific filtering

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#2060]
Please note: the FAIRsharing Assistant is the current priority, therefore this has been marked as low priority and doesn't need to be done until the initial version of the Assistant is complete.

The new relationship graphs are fantastic! I've had quite a few people comment on them, and everyone seems very happy.

I would like to have a change in how the filtering works, please: I would like to be able to filter by registry type, independently, at each "hop" level.

This would allow, for example, a collection graph to show just the collected databases in the "one hop", then all standards in the "two hop", and all databases in the "three hop". The result would be that we would see only those standards that are directly linked to the collected databases, and then be able to explore **all** databases that are directly linked to those standards.

As another example, let's say we're starting with GenBank, and I'd like to find out all of the databases that are linked to any standard that is directly linked to GenBank. I'd set the "one hop" to standards, then the "two hop" to databases. Does that make sense?

Thanks!

**Comment by allysonlister** [comment]
Please note that this ticket is still active and development should continue. It was noted, however, that this is not a trivial (or even necessarily possible) task for the current graph library that is in use, and therefore a solution may take some time.
